src/k8s/go.mod

```diff
@@ -1,31 +1,31 @@
 module github.com/canonical/k8s
 
-go 1.22.6
+go 1.22.7
 
 require (
 	dario.cat/mergo v1.0.0
-	github.com/canonical/go-dqlite v1.22.0
+	github.com/canonical/go-dqlite/v3 v3.0.0
 	github.com/canonical/k8s-snap-api v1.0.13
-	github.com/canonical/lxd v0.0.0-20240822122218-e7b2a7a83230
-	github.com/canonical/microcluster/v3 v3.0.0-20240827143335-f7a4d3984970
+	github.com/canonical/lxd v0.0.0-20241106103908-9ac2433510c8
+	github.com/canonical/microcluster/v3 v3.0.0-20241106092713-7f54bbd37f6f
```

Comment:
Just making sure that I understand this: 
We were using a commit from `microcluster v3` that used `go-dqlite v1`, but now, we switching to a newer commit of `microcluster v3` which uses `go-dqlite v3`, right?

Reply:
That's correct.